barman/Dockerfile.template

```diff
@@ -6,7 +6,10 @@ RUN apt-get update \
 		rsync \
 		supervisor \
 	&& rm -Rf /var/lib/apt/lists/* \
-	&& sed -i 's/\(.*pam_loginuid.so\)/#\1/' /etc/pam.d/sshd
+	&& sed -i 's/\(.*pam_loginuid.so\)/#\1/' /etc/pam.d/sshd \
+	&& mkdir /home/postgres \
```

Comment:
Instead of manually doing these steps, just  add a `-m` argument to the `useradd` command in Dockerfile-debian.template.  That should cause /home/postgres to get created with the correct ownership/permissions.